pkg/config/templates/wicked-ifcfg-vlan

```diff
@@ -0,0 +1,13 @@
+STARTMODE='onboot'
+BOOTPROTO='{{ .Method }}'
```

Comment:
Shouldn't it be `.Network.Method`? Or use `with` action for convenience.

Reply:
I update the struct like the following, so we can keep wicked files structure same.

```
struct {
  Name string
  Network
}
```